TestsExample/App.js

```diff
@@ -6,8 +6,9 @@ import Test1374 from './src/Test1374';
 import Test1718 from './src/Test1718';
 import Test1813 from './src/Test1813';
 import Test1845 from './src/Test1845';
+import Test2080 from './src/Test2080';
 import PointerEventsBoxNone from './src/PointerEventsBoxNone';
 
 export default function App() {
-  return <ColorTest />;
+  return <Test2080 />;
```

Comment:
We don't want to change the default test here.

```suggestion
  return <ColorTest />;
```